### ISSUE TITLE
Feature idea: Character sub menu

### ISSUE THREAD

**Issue by johnrose433** [Eusth/PlayClubVR#1]
Taking your forum advice and posting some feature ideas. Absolutely stunning work, thanks so much for the amazing mod.

Would be incredible to have a separate controller menu for "character":

Circular thumb-pad controls clothing state:
Press up removes top
Press down removes bottom
Press left removes bra
Press right removes panties

Trigger cycles through custom characters:
Each trigger pull automatically loads the next alphabetical character in your directory of PNGs

**Comment by Eusth** [comment]
I like the idea of making a tool for changing the character state.

I wouldn't use the trigger for switching through the characters, though, since it's the button that is easiest pressed and it's hard to get back to your previous character. Perhaps it could be used in conjunction with the thumbpad, à la you can back and forth by pressing left or right while pressing the trigger.

**Comment by johnrose433** [comment]
Yeah, that actually sounds way better since the trigger/pad combo would allow you to cycle both forward and reverse through all the characters. Great stuff, thanks for the reply.
 

    On Tuesday, May 31, 2016 2:40 PM, Eusth <notifications@github.com> wrote:
 

 I like the idea of making a tool for changing the character state.I wouldn't use the trigger for switching through the characters, though, since it's the button that is easiest pressed and it's hard to get back to your previous character. Perhaps it could be used in conjunction with the thumbpad, à la you can back and forth by pressing left or right while pressing the trigger.—

**Comment by johnrose433** [comment]
Do you think this would be a possible addition to your next release? Changing through characters without having to drill down into the sub-menus would be great (and having quick access to clothing state). Regardless, thanks for all of your excellent work.
